firebase-installations-interop/src/main/java/com/google/firebase/installations/FirebaseInstallationsApi.java

```diff
@@ -47,15 +46,18 @@
    * Async function that returns a globally unique identifier of this Firebase app installation.
    * This is a url-safe base64 string of a 128-bit integer.
    */
+  @NonNull
   Task<String> getId();
 
   /** Async function that returns a auth token(public key) of this Firebase app installation. */
+  @NonNull
   Task<InstallationTokenResult> getToken(@AuthTokenOption int authTokenOption);
```

Comment:
@vkryachko Should the class members also be public to be documented on firebase site?

Reply:
Yes, and since this is an interface, all of its methods are public. Also note my other comment on this interface, I don't think it should be documented

Reply:
Thanks. Addressed.